contrib_bots/lib/github_detail.md

```diff
@@ -0,0 +1,27 @@
+# GitHub detail bot
+This bot links and details issues and pull requests.
+To use it username mention the bot then type an id:
```

Comment:
I think we should make it clear in the code and in the docs, e.g. [this line](https://github.com/zulip/zulip/pull/2881/files#diff-0d16bbf5fd38bb3ac2bb8bf55f995604R39) is confusing at the moment. 

Could you use `<bot_name>` or sth similar as a placeholder in the docs and the bot script?